docs/safe-haven-services/tre-container-user-guide/development-workflow.md

```diff
@@ -321,45 +321,27 @@ The test environment is located in the eidf147 project. Please ask your research
 
 Containers can only be used on the TRE desktop hosts using shell commands. Containers can only be pulled from the GHCR into the TRE using a `ces-pull` script. Hence containers must be pushed to GHCR for them to be used in the TRE. Although alternative methods can be used in the test environment, we encourage users to follow the exact same procedure as they would in the TRE.
 
-Once access has been granted to the test environemnt in the eidf147 project, the user can pull a container from their private GHCR repository using the command:
+Once access has been granted to the test environemnt in the eidf147 project, the user can pull a container from their private GHCR repository using the `ces-pull` command:
 
 ```sh
-ces-pull <container-engine> <github_user> <github_token> ghcr.io/<namespace>/<container_name>:<container_tag>
+ces-pull [<runtime>] <github_user> <ghcr_token> ghcr.io/<namespace>/<container_name>:<container_tag>
 ```
 
-> [!NOTE]
```

Comment:
This was previously broken, and I'm not sure if an admonition is needed since clarifications on the other CLI options don't use one and adding them for all would clutter the document.